specs/Freeze.md

```diff
@@ -0,0 +1,155 @@
+# Background
+Edge browser has a sleeping tab feature which reduces resource usage when the tab is in the background. We are introducing WebView2 APIs
+to access this feature so that invisible WebView can use less resources. We'd appreciate your feedback.
+
+
+# Description
+You may call the `TryFeezeAsync` API to have the WebView2 consume less memory. This is useful when your Win32 app becomes invisible, or when your Universal Windows Platform app is suspended, during the suspended event handler before completing the suspended event.
+
+# Examples
+## .Net, WinRT
+```c#
+async protected void OnSuspending(object sender, SuspendingEventArgs args)
+{
+    SuspendingDeferral deferral = args.SuspendingOperation.GetDeferral();
```

Comment:
Don't explicitly call deferral.Complete, instead use using:

```cs
using(args.SuspendingOperation.GetDeferral())
{
    webView.Visibility = false;
    await webView.CoreWebView2.TryFreezeAsync();
}
```

Reply:
Leave the sample code as is.